searchsummary/src/tests/docsummary/copy_dfw/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+vespa_add_executable(searchsummary_copy_dfw_test_app TEST
```

Comment:
I think there is a new CMake function to add a test now... (not having to write the name 3 times)